extensions/amp-render/1.0/component.js

```diff
@@ -110,7 +110,7 @@ export function RenderWithRef(
     rendered && typeof rendered == 'object' && '__html' in rendered;
 
   return (
-    <Wrapper {...rest} dangerouslySetInnerHTML={isHtml ? rendered : null}>
+    <Wrapper {...rest} dangerouslySetInnerHTML={isHtml ? rendered : null} aria-live="polite">
```

Comment:
Please propagate if an `aria-live` value is passed through props and add unit tests for this layer.

Reply:
Done